src/components/Legend/Legend.styles.js

```diff
@@ -18,28 +18,28 @@ function rotationBuilder() {
 }
 
 export const LegendUseclass = styled.div.attrs({
-  className: 'fixed left-0 bottom-0 z-max'
+  className: 'fixed left-0 bottom-0 z-max',
 })``;
 
 export const Circle = styled.div.attrs({
-  className: 'mt2 w2 h2 br-100 flex items-center justify-center '
+  className: 'mt2 w2 h2 br-100 flex items-center justify-center ',
 })`
   background: ${props => props.color};
 `;
 
 export const AnimateCircle = styled.div.attrs({
-  className: 'mt2 w2 h2 br-100 flex items-center justify-center '
+  className: 'mt2 w2 h2 br-100 flex items-center justify-center ',
 })`
   background: ${props => props.color};
   animation: ${rotationBuilder()} 0.5s linear;
 `;
 
 export const UseText = styled.p.attrs({
-  className: 'avenir black'
+  className: 'avenir black',
```

Comment:
I think it may be to do with importing the antd css file to the frontend, but now the keys for use class are not vertically centered. There is a margin on the `p` element that contains the text

<img width="72" alt="Screen Shot 2019-09-25 at 11 28 19" src="https://user-images.githubusercontent.com/28838993/65593187-e362e900-df87-11e9-90e0-f0ba4b861287.png">

Can be resolved by adding the class `ma0` to the UseText component